test/allreduce_torchrun.py

```diff
@@ -0,0 +1,29 @@
+import torch
```

Comment:
test won't get run in CI unless you added it to https://github.com/pytorch/xla/blob/master/test/run_tests.sh#L151

Reply:
Sounds good! will add it here